bait-files/src/main/webapp/review.css

```diff
@@ -0,0 +1,127 @@
+a {
+  text-decoration: none;
+}
+
+.accordion {
+  background-color: #ffffff;
+  border: none;
+  border-radius: 8px;
+  color: #202124;
+  cursor: pointer;  
+  font-size: 18px;
+  font-weight: 500;
+  margin: 8px 0px 0px 0px;
+  outline: none;
+  padding: 18px;
+  transition: 0.4s;
+  text-align: left;
+  width: 100%;
+}
+
+.accordion:after {
+  color: #202124;
+  content: '+';
+  float: right;
+  font-weight: 500;
+}
+
+.active, .accordion:hover {
+  background-color: #dadce0;
+}
+
+.active {
+  background-color: #ffffff;
+  border-bottom-left-radius: 0px;
+  border-bottom-right-radius: 0px;
+}
+.active:after {
+  content: "-";
+}
+
+body {
+  background-color: #e8eaed;
+  color: #3c4043;
+  display: flex;
+  flex-direction: column;
+  font-family: 'Roboto', sans-serif;
+  margin-top: 0;
+}
+
+button{
+  background-color: #bdc1c6;
+  border-radius: 8px;
+  border: none;
+  font-size: 14px;
+  margin: 15px 0px;
+  outline: none;
+  padding: 6px 8px;
+}
+
+.button-container{
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+}
+
+content {
+  align-items: center;
+  display:flex;
+  flex-direction: column;
+}
+
+#review-overview, nav {
```

Comment:
Do you really want to include nav here?  It looks like you are defining all four of the properties here below as well (in the next block down).